@navikt/core/react/src/util/virtualfocus/parts/VirtualFocusContent.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { forwardRef } from "react";
+import { useVirtualFocusInternalContext } from "../VirtualFocus";
+
```

Comment:
<img width="622" alt="Screenshot 2024-06-27 at 13 39 02" src="https://github.com/navikt/aksel/assets/26967723/facb52c2-2474-42ec-b9bc-4e3b3a1168ab">

Github detects cyclic imports now 🤯

Reply:
neat! 🚀

Reply:
or actually, I think it's just forwarding the string output from our lint step? 🤔 maybe.